Pilot Manuscript scripts/NMF Clustering/NMF cophenetic.Rmd

```diff
@@ -0,0 +1,122 @@
+---
+title: "PTRC Ex10 Batch correction and normalization"
+author: "Michael Nestor"
+date: "`r format(Sys.Date(), '%m/%d/%Y')`"
+output: html_document
+---
+
+```{r setup}
+library(MSnSet.utils)
+library(NMF)
+library(dplyr)
+library(tibble)
+library(tidyr)
+library(ggplot2)
+
+
+source("../../util/synapseUtil.R")
+source("../../util/loading_data.R")
+source("NMF helper.R")
```

Comment:
Can you change the name of this directory to 'analysis/NMF' it is best to avoid spaces in paths and also it's not the pilot manuscript...